playwright/e2e/crypto/staged-rollout.spec.ts

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2024 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { test, expect } from "../../element-web-test";
+import { logIntoElement } from "./utils";
+
+test.describe("Migration of existing logins", () => {
+    test("Test migration of existing logins when rollout is 100%", async ({
+        page,
+        context,
+        app,
+        credentials,
+        homeserver,
+    }, workerInfo) => {
+        test.skip(workerInfo.project.name === "Rust Crypto", "This test only works with Rust crypto.");
```

Comment:
We want to test legacy to rust migration (feature_rust_crypto from `false` to `true`). 
In the project `Rust Crypto`, the app is started with `feature_rust_crypto: true`
But anyhow it doesn't matter much as we anyhow override the `cryptoBackend` by intercepting `http://localhost:8080/config.json*`, so it would test twice the same thing.